google/cloud/bigquery/_helpers.py

```diff
@@ -382,7 +378,16 @@ def _field_to_index_mapping(schema):
 
 
 def _field_from_json(resource, field):
-    converter = _CELLDATA_FROM_JSON.get(field.field_type, lambda value, _: value)
+    def default_converter(value, field):
```

Comment:
Maybe we can move this function to outside, so we can reuse it for `_scalar_field_to_json()` as well?

Reply:
I ended up just moving the warning itself to a new function, since the converter in _scalar_field_to_json has a different function signature.